### PR TITLE
Fix event loop test

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Comment by lazynina** [comment]
* **#1159** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1159?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lazynina and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: consensus/event_loop_test.go [hunk 1]
```diff
@@ -156,7 +156,7 @@ func TestProcessTipBlock(t *testing.T) {
 	// Test ProcessTipBlock() function when event loop is not running
 	{
 		tipBlock := BlockWithValidatorList{createDummyBlock(2), createDummyValidatorList()}
-		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock})
+		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock}, time.Second, time.Second)
```
Comment (tholonious) [inline consensus/event_loop_test.go:159]:
```suggestion
		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock}, oneHourInNanoSecs, oneHourInNanoSecs)
```

File: consensus/event_loop_test.go [hunk 2]
```diff
@@ -168,6 +168,8 @@ func TestProcessTipBlock(t *testing.T) {
 		err := fc.ProcessTipBlock(
 			BlockWithValidatorList{nil, createDummyValidatorList()},                     // tip
 			[]BlockWithValidatorList{{createDummyBlock(2), createDummyValidatorList()}}, // safeBlocks
+			time.Second,
+			time.Second,
```
Comment (tholonious) [inline consensus/event_loop_test.go:172]:
```suggestion
			oneHourInNanoSecs, oneHourInNanoSecs,
```

File: consensus/event_loop_test.go [hunk 3]
```diff
@@ -177,6 +179,8 @@ func TestProcessTipBlock(t *testing.T) {
 		err := fc.ProcessTipBlock(
 			BlockWithValidatorList{createDummyBlock(2), nil},                            // tip
 			[]BlockWithValidatorList{{createDummyBlock(2), createDummyValidatorList()}}, // safeBlocks
+			time.Second,
+			time.Second,
```
Comment (tholonious) [inline consensus/event_loop_test.go:183]:
```suggestion
			oneHourInNanoSecs, oneHourInNanoSecs,
```

File: consensus/event_loop_test.go [hunk 4]
```diff
@@ -186,6 +190,8 @@ func TestProcessTipBlock(t *testing.T) {
 		err := fc.ProcessTipBlock(
 			BlockWithValidatorList{createDummyBlock(2), createDummyValidatorList()}, // tip
 			[]BlockWithValidatorList{{nil, createDummyValidatorList()}},             // safeBlocks
+			time.Second,
+			time.Second,
```
Comment (tholonious) [inline consensus/event_loop_test.go:194]:
```suggestion
			oneHourInNanoSecs, oneHourInNanoSecs,
```

File: consensus/event_loop_test.go [hunk 5]
```diff
@@ -195,6 +201,8 @@ func TestProcessTipBlock(t *testing.T) {
 		err := fc.ProcessTipBlock(
 			BlockWithValidatorList{createDummyBlock(2), createDummyValidatorList()}, // tip
 			[]BlockWithValidatorList{{createDummyBlock(2), nil}},                    // safeBlocks
+			time.Second,
+			time.Second,
```
Comment (tholonious) [inline consensus/event_loop_test.go:205]:
```suggestion
			oneHourInNanoSecs, oneHourInNanoSecs,
```

File: consensus/event_loop_test.go [hunk 6]
```diff
@@ -252,7 +260,7 @@ func TestProcessTipBlock(t *testing.T) {
 
 		tipBlock := BlockWithValidatorList{nextBlock, createDummyValidatorList()}
 
-		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock})
+		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock}, time.Second, time.Second)
```
Comment (tholonious) [inline consensus/event_loop_test.go:263]:
```suggestion
		err := fc.ProcessTipBlock(tipBlock, []BlockWithValidatorList{tipBlock}, oneHourInNanoSecs, oneHourInNanoSecs)
```

**Review by tholonious — APPROVED** [review]

**Pull request merged**